nomad/acl_endpoint.go

```diff
@@ -2761,14 +2774,38 @@ func (a *ACL) OIDCCompleteAuth(
 		return err
 	}
 
+	// No need to do all this marshaling if VerboseLogging is disabled
+	if authMethod.Config.VerboseLogging {
+		idTokenClaimBytes, err := json.MarshalIndent(idTokenClaims, "", " ")
+		if err != nil {
+			return fmt.Errorf("failed to marshal ID token claims")
+		}
+
+		userClaimBytes, err := json.MarshalIndent(userClaims, "", " ")
+		if err != nil {
+			return fmt.Errorf("failed to marshal user claims")
+		}
+
+		internalClaimBytes, err := json.MarshalIndent(oidcInternalClaims.List, "", " ")
+		if err != nil {
+			return fmt.Errorf("failed to marshal OIDC internal claims list")
+		}
+
+		vlog.Debug("Claims",
+			"Token Claims", string(idTokenClaimBytes),
+			"User Claims", string(userClaimBytes),
+			"Claims after mapping to Nomad identity attributes", string(internalClaimBytes),
+		)
```

Comment:
Could we update this, to make the log line a little more descriptive and lowercase, and ensure the context keys use our standard snake_case format?